src/writeckpt.cpp

```diff
@@ -433,7 +441,15 @@ static void writememoryarea (int fd, Area *area, int stack_was_seen)
      *   implemented with backing files
      */
     JASSERT((area->flags & MAP_ANONYMOUS) || (area->flags & MAP_SHARED));
-    Util::writeAll(fd, area, sizeof(*area));
-    Util::writeAll(fd, area->addr, area->size);
+
+    if (skipWritingTextArea && (area->prot & PROT_EXEC)) {
+      area->properties = DMTCP_SKIP_WRITING_TEXT_AREA;
+      Util::writeAll(fd, area, sizeof(*area));
+      JTRACE("Skipping over text-area") (area->name) ((void*)area->addr);
```

Comment:
text-area -> text segment memory

[ First, avoiding the hyphen; second, the JTRACE should use standard terminology ("text segment memory"), while the reader can quickly translate this into our internal variables with the terminology "area".  If you object to this, my fall-back suggestion is "text area in memory" ]